node/src/main/java/eu/cloudnetservice/node/util/NetworkUtil.java

```diff
@@ -68,6 +75,86 @@ public static boolean isInUse(int port) {
     }
   }
 
+  public static @Nullable HostAndPort assignableHostAndPort(@NonNull HostAndPort hostAndPort) {
+    try (var socket = new ServerSocket()) {
+      // try to bind on the given address
+      socket.bind(new InetSocketAddress(hostAndPort.host(), 45555));
```

Comment:
```suggestion
      socket.bind(new InetSocketAddress(hostAndPort.host(), 0));
```
Should bind the socket to the next free port